.github/workflows/rspec_tests.yaml

```diff
@@ -0,0 +1,38 @@
+---
+name: RSpec test
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  rspec_tests:
+    name: ${{ matrix.cfg.os }}(ruby ${{ matrix.cfg.ruby }})
+    strategy:
+      matrix:
+        cfg:
+          - {os: ubuntu-18.04, ruby: '2.5'}
+          - {os: ubuntu-18.04, ruby: '2.7'}
+          - {os: windows-2019, ruby: '2.5'}
+          - {os: windows-2019, ruby: '2.7'}
+
+    runs-on: ${{ matrix.cfg.os }}
+    steps:
+      - name: Checkout current PR
+        uses: actions/checkout@v2
+
+      - name: Install ruby version ${{ matrix.cfg.ruby }}
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.cfg.ruby }}
+
+      - name: Update rubygems and install gems
+        run: |
+          gem update --system --silent --no-document
+          bundle config set without packaging development
+          bundle install --jobs 4 --retry 3
+      
+      - run: "bundle exec rake spec"
+      - run: "bundle exec rake commits"
```

Comment:
Sorry just saw this, I think we can drop the `rake commits`, since it's in `checks.yaml`

Reply:
Oh heck, I was going to remove that and just forgot, I'll change that now!